module/TaskManagement/test/integration/BaseTaskProcessTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+use IntegrationTest\Bootstrap;
+use Zend\Http\Request;
+use Zend\Http\Response;
+use Zend\Mvc\MvcEvent;
+use Zend\Mvc\Router\Http\TreeRouteStack as HttpRouter;
+use Zend\Mvc\Router\RouteMatch;
+use Zend\Uri\Http;
+
+/**
+ * Created by PhpStorm.
+ * User: rad
+ * Date: 08/03/17
+ * Time: 19:54
+ */
+class BaseTaskProcessTest extends \PHPUnit_Framework_TestCase
+{
+    protected $controller;
+    protected $event;
+    protected $request;
+    protected $response;
+    protected $routeMatch;
+    protected $serviceManager;
+    protected $taskService;
+    protected $streamService;
+    protected $userService;
+    protected $organizationService;
+    protected $kanbanizeService;
+
+    public function __construct()
+    {
+        $this->serviceManager = Bootstrap::getServiceManager();
+        $this->userService = $this->serviceManager->get('Application\UserService');
+        $this->taskService = $this->serviceManager->get('TaskManagement\TaskService');
+        $this->streamService = $this->serviceManager->get('TaskManagement\StreamService');
+        $this->organizationService = $this->serviceManager->get('People\OrganizationService');
+        $this->kanbanizeService = $this->serviceManager->get('Kanbanize\KanbanizeService');
+    }
+
+    protected function setupController($controller, $action)
```

Comment:
i controller sono definiti come servizi, puoi fare in modo che il serviceLocator li istanzi per te, in modo da non dover passare tutte le dipendenze